data/tests.yml

```diff
@@ -1080,39 +1082,67 @@
     designated: motor_vehicle
 
 - way_id: 898731283
-  comment: how should an odd number of lanes forward/backwards be split without any clues?
+  comment: guess that an odd number of lanes will be divided favouring the forward direction with a warning
+  rust:
+    expect_warnings: true
   tags:
     highway: 'road'
     lanes: '3'
     sidewalk: 'both'
   driving_side: 'left'
   output:
-  - type: travel
-    designated: foot
-  - type: travel
-    direction: forward
-    designated: motor_vehicle
-  - type: travel
-    direction: forward
-    designated: motor_vehicle
-  - type: travel
-    direction: backward
-    designated: motor_vehicle
-  - type: travel
-    designated: foot
+    - type: travel
+      designated: foot
+    - type: travel
+      direction: forward
+      designated: motor_vehicle
+    - type: travel
+      direction: forward
+      designated: motor_vehicle
+    - type: travel
+      direction: backward
+      designated: motor_vehicle
+    - type: travel
+      designated: foot
```

Comment:
given the new indenting, refactor to the new test scheme:

```
  road:
    highway: road
    lanes:
    - type: travel
      designated: foot
    - type: travel
      direction: forward
      designated: motor_vehicle
    - type: travel
      direction: forward
      designated: motor_vehicle
    - type: travel
      direction: backward
      designated: motor_vehicle
    - type: travel
      designated: foot
```

Reply:
note the one additional layer of nesting.

it saves me doing it later :)

this applies to the other cases as well

Reply:
I think I have updated all the tests that will produce merge conflicts within the `output` lines.